dialects/linux/dproc.c

```diff
@@ -1395,6 +1397,57 @@ process_id(idp, idpl, cmd, uid, pid, ppid, pgid, tid, tcmd)
 	return(0);
 }
 
+/* compare namespace of this lsof proces and the target process */
+
+int
+compare_namespace(ns)
```

Comment:
How about passing `pid`, instead of referring `Lp->pid`?

Reply:
Helpful, passing pid is more flexible.